File: android/src/main/java/expo/modules/settings/ExpoSettingsModule.kt
package expo.modules.settings

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class ExpoSettingsModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("ExpoSettings")

    Function("get") { key: String ->
      return@Function "system"
    }

    Function("set") { key: String, value: String ->
    }

    Function("remove") { key: String ->
    }
  }
}

File: ios/ExpoSettingsModule.swift
import ExpoModulesCore

public class ExpoSettingsModule: Module {
  public func definition() -> ModuleDefinition {
    Name("ExpoSettings")

    Function("get") { (key: String) -> String in
      "100"
    }

    Function("set") { (key: String, value: String) in
    }

    Function("remove") { (key: String) in
    }
  }
}
